keel-sql/src/main/resources/db/changelog/20200302-drop-api-version.yml

```diff
@@ -0,0 +1,16 @@
+databaseChangeLog:
+- changeSet:
+    id: drop-api-version
+    author: fletch
+    changes:
```

Comment:
This migration doesn't have a rollback in it. What's the convention for this project for specifying rollbacks?

Reply:
I usually add one but I noticed the previous couple of migrations didn’t have them